build/sir/Dockerfile

```diff
@@ -11,18 +11,14 @@ LABEL org.label-schema.schema-version="1.0.0-rc1" \
 # From metabrainz/sir #
 #######################
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-                    build-essential \
-                    ca-certificates \
-                    cron \
-                    git \
-                    libpq-dev \
-                    libffi-dev \
-                    libssl-dev \
-                    libxml2-dev \
-                    libxslt1-dev && \
```

Comment:
This comes from sir Dockerfile but I did not find any reason why these would be needed. Since it requires further live testing, let’s try removing these in a later PR.

Reply:
I don't see a reason why we should keep it then but I'll open a new PR with it removed. Better safe then sorry.

Reply:
`libxml2-dev` and `libxslt1-dev` are required by `lxml` which is installed with `mb-rngpy`.

Still looking for `libffi-dev` and `libssl-dev`.

Reply:
``libxslt1-dev`` has a dependency on ``libxml2-dev`` so we could drop that.
I could imagine ``libffi-dev`` is needed for a python package which compiles C code.

Reply:
> `libxslt1-dev` has a dependency on `libxml2-dev` so we could drop that.

Not sure it would improve anything but making harder to check dependencies.

>  I could imagine `libffi-dev` is needed for a python package which compiles C code.

Either to install a Python package or to use `ctypes` in [`sir/indexing.py`](https://github.com/metabrainz/sir/blob/v1.0.2/sir/indexing.py).

I could imagine `libssl-dev` might be required too as `sir` opens a lot of connections through a variety of Python packages.

Reply:
I put `libssl-dev` back in.

Reply:
Finally, `libffi-dev` seems safe to remove, see https://github.com/metabrainz/musicbrainz-docker/pull/123#pullrequestreview-371835081.